src/Groups/perm.jl

```diff
@@ -149,6 +149,18 @@ function perm(L::AbstractVector{<:IntegerUnion})
   return PermGroupElem(symmetric_group(length(L)), GAPWrap.PermList(GapObj(L;recursive=true)))
 end
 
+"""
+  smaller_degree_permutation_representation(G::PermGroup) -> PermGroup, map
+  
+Return an isomorphic permutation group of smaller degree and the isomorphism.
+"""
+function smaller_degree_permutation_representation(G::PermGroup)
+     mp = GAP.Globals.SmallerDegreePermutationRepresentation(GapObj(G))
+     img = PermGroup(GAP.Globals.Image(mp))
+     return img, GAPGroupHomomorphism(G, img, mp)
+end
+
+simplify(G::PermGroup) = smaller_degree_permutation_representation(G)
```

Comment:
Why both names? Let's pick one?

Reply:
`simplify` does not have a specified behavior. It returns whatever a developer considers "simpler" than the input. Would this for instance include reducing the size of a generating set using `small_generating_set`?

But then `simplify` is nice for non-experts who are not aware which kinds of simplifications could be useful.

Reply:
As far as I understand, `simplify` is defined for various objects, and user who know about its existence may be tempted to call `simplify` with a group. I think this makes sense only if it is documented in which respect the return value may differ from the input.

For example, it makes a difference whether `simplify` is allowed to return an isomorphic group (that acts in a different way on sets on which the input group acts) or whether `simplify` returns "the same" group (in the sense that it has the same elements) but a "nicer" set of generators.

It is important that `simplify` does not change its input in place, i.e., there should never be a `simplify!` for groups.

Reply:
I would say that the only requirement on simplify is to return an isomorphic object plus the isomorphism.
This could be a global definition of simplify for any kind of parent object. 

But if this is controversial, I can just delete it from this PR.